app/helpers/application_helper.rb

```diff
@@ -32,22 +54,41 @@ def euro_from_cents(f)
     end
   end
 
+  # Convert a given float in euro's to a formatted string.
   def euro(f)
-    number_to_currency(f, unit: '€')
+    number_to_currency(f, unit: "€")
+  end
+
+  # Replace all instances of double quotes with single quotes
+  # This can be useful for escaping strings in HTML
+  def escape_quotes(str)
+    str.gsub("\"", "'")
   end
 
+  # Create a formatted form.
   def f_form_for(record, options = {}, &block)
     options[:builder] = FormattedFormBuilder
     form_for(record, options, &block)
   end
 
+  # Get the current theme of the user
+  def theme
+    {
+      variant: {
+        name: cookies[:themeVariantName],
+        hue: cookies[:themeVariantHue]
+      },
+      mode: cookies[:themeMode] || "light"
+    }
+  end
+
   def slack_notification(user, message)
-    require 'net/http'
-    require 'json'
-    postData = Net::HTTP.post_form(URI.parse('https://slack.com/api/users.list'), {'token'=>Rails.application.secrets.access_token})
+    require "net/http"
+    require "json"
+    postData = Net::HTTP.post_form(URI.parse("https://slack.com/api/users.list"), { "token" => Rails.application.secrets.access_token })
```

Comment:
Huh, ik wist niet dat dit een ding is. Wat doet dit? Mss eens kijken om naar MM te porten?